scripts/rc.py

```diff
@@ -125,6 +139,19 @@ def delete_ts(ts,enc_name):
     cmd = subprocess.Popen([ts, "purge", "-i", ts_name], stdin=PIPE)
     cmd.communicate(input='y')
 
+def ts_start_server(ts, listener):
+    logger.debug(listener)
+    ts_parameters = [ts, "runserver", "-h", listener] 
+    logger.debug(ts_parameters)
+    cmd = subprocess.Popen(ts_parameters, stdout=subprocess.PIPE)
+    logger.info("Timesketch Running pid: {} - listening on {}".format(cmd.pid, listener))
+
+def ts_stop_server():
+    logger.debug("Attempting to stop timesketch server")
+    logger.info("Stopping TimeSketch Service")
+    cmd = subprocess.call("killall tsctl", shell=True, stdout=subprocess.PIPE)
```

Comment:
For all subprocess.call calls can we add a bit to check if the output was non 0 and print it out if it was